packages/@okta/vuepress-site/docs/reference/rl-clientbased/index.md

```diff
@@ -8,25 +8,27 @@ excerpt: >-
 
-Currently, the client-based rate limits apply to the `/login/login.htm` endpoint and OAuth 2.0 `/authorize` endpoint on both the Okta Org Authorization Server and any Custom Authorization Server. All Custom Authorization Servers share a [rate limit](/docs/reference/rate-limits/). The Org Authorization Server has a separate rate limit.
+Currently, the client-based rate limits apply to the `/login/login.htm` endpoint and OAuth 2.0 `/oauth2/v1/authorize` endpoint on both the Okta Org Authorization Server and any Custom Authorization Server. All Custom Authorization Servers share a [rate limit](/docs/reference/rate-limits/). The Org Authorization Server has a separate rate limit.
 
-Each valid request made by a user to this endpoint is counted as one request against the respective [authorization server](/docs/concepts/auth-servers/) rate limit bucket, for example, (`/oauth2/${authorizationServerId}/v1` (Custom Authorization Server) or `/oauth2/v1` (Okta Org Authorization Server). The per minute rate limits on these endpoints apply across an Okta tenant.
+Each valid request made by a user to this endpoint is counted as one request against the respective [authorization server](/docs/concepts/auth-servers/) rate limit bucket, for example, `/oauth2/${authorizationServerId}/v1` (Custom Authorization Server) or `/oauth2/v1` (Okta Org Authorization Server). The per minute rate limits on these endpoints apply across an Okta tenant.
 
-For example, example.com has 10 OAuth 2.0 applications running in a production environment. Bob's team is launching a new marketing portal that is a single page OAuth 2.0 application. Unaware of the rate limits on the `/authorize` endpoint, Bob's team begins running some batch testing scripts against the newly created application that makes hundreds of `/authorize` requests in a single minute. Without the client-based rate limit framework, the new marketing portal application could potentially consume all of the per minute request limits assigned to example.okta.com and thereby cause rate limit violations for the rest of the users that access the other OAuth 2.0 applications.
+Curently, you cannot update the per client rate limit. Every client ID/IP/device identifier combination (for the /authorize endpoint) or IP/device identifier combination (for the /login/login.htm endpoint) gets 60 total requests per minute and a maximum of two concurrent requests.
```

Comment:
Should the endpoints in this paragraph be in monospace?

Reply:
good catch! fixed.